subsys/net/ip/net_pkt.c

```diff
@@ -1823,6 +1823,8 @@ bool net_pkt_is_contiguous(struct net_pkt *pkt, size_t size)
 	if (pkt->cursor.buf && pkt->cursor.pos) {
```

Comment:
There is some garbage at the end of commit message of "net: ip: net_pkt: Add pkt_cursor_advance before contiguous check"
Also there is redundant information in the commit subject, and could be written shorter like this "net: pkt: Add pkt_cursor_advance before contiguous check"

Reply:
Removed.